sqlflow_models/dbscan.py

```diff
@@ -0,0 +1,179 @@
+#!usr/bin/env python
+# -*- coding:utf-8 _*-
+
+"""
+__author__ : tiankelang
+__email__ : kelang@mail.ustc.edu.cn
+__file_name__ : dbscan.py
+__create_time__ : 2020/07/01
+"""
+import numpy as np
+import tensorflow as tf
+from scipy.spatial.distance import pdist, squareform
+from sklearn.base import BaseEstimator, ClusterMixin
+import pandas as pd
+from sklearn import datasets, metrics
+
+def optimizer():
```

Comment:
It seems that it's not a NN model, should we remove this function?

Reply:
Yes, there is no need to set optimizer.